python-test/features/steps/control_plane_policies.py

```diff
@@ -42,6 +50,52 @@ def new_policy(context):
     check_policies(context)
 
 
+@step('the container logs contains the message "{text_to_match}" referred to all applied policies within {'
+      'time_to_wait} seconds after applying all of them')
```

Comment:
Looks like we're talking about 180 seconds after the policies have been applied, not logs messages printed after policies have been applied.